src/cmd/tools/helm/load_plugins.go

```diff
@@ -318,11 +318,14 @@ func loadFile(path string) (*pluginCommand, error) {
 	cmds := new(pluginCommand)
 	b, err := os.ReadFile(path)
 	if err != nil {
-		return cmds, fmt.Errorf("file (%s) not provided by plugin. No plugin auto-completion possible", path)
+		return &pluginCommand{}, fmt.Errorf("file (%s) not provided by plugin. No plugin auto-completion possible", path)
```

Comment:
I think it is better to return nil instead of a pointer to an empty struct.

Reply:
Yeah I agree with this, I usually do that but I'd seen some empty declarations in various places.